backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:slim-buster
+
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get -y install libpq-dev
+
+WORKDIR /app
+COPY . /app/
+
+EXPOSE 7777
+
+
+ENTRYPOINT ["/bin/bash", "-c", "cargo run --release"]
```

Comment:
wtf?

Reply:
Since albi insists on properly formulated review comments: Why did you choose to not build the binary and put it into the container, but rather to recompile the binary on every container start?

Reply:
This was a debugging artifact. I was attempting to debug why my build process didn't work with sqlx. Therefore, I had this minimalistic Docker file to quickly make changes and test things. Ideally, I want to have a build process in my docker file where I compile the code and move the binary into another container. If this does not work, I will follow the approach you did by simple copy the binary from my home system to the container.

Reply:
Don’t copy the executable from the host. It will have your home directory in error messages etc.